src/trading/ShareToken.sol

```diff
@@ -49,4 +49,11 @@ contract ShareToken is DelegationTarget, Typed, Initializable, VariableSupplyTok
     function isShareToken() constant public returns(bool) {
         return true;
     }
+
+    function transfer(address _to, uint256 _value) public returns(bool) {
```

Comment:
This implementation looks identical to the one on `BasicToken` (which is a base contract for `ShareToken`).  What is the reason for the override here?

Reply:
Woops. This was left in from debugging. Will remove.